lca-cli/postpivot/postpivot.go

```diff
@@ -225,6 +230,48 @@ func (p *PostPivot) recert(ctx context.Context, seedReconfiguration *clusterconf
 	return nil
 }
 
+func (p *PostPivot) setProxyAndProxyStatus(seedReconfig *clusterconfig_api.SeedReconfiguration, seedClusterInfo *seedclusterinfo.SeedClusterInfo) error {
+	if seedReconfig.Proxy != nil != seedClusterInfo.HasProxy {
+		return fmt.Errorf("seedReconfig and seedClusterInfo proxy configuration mismatch")
```

Comment:
LGTM from me! 

But on the topic of mismatch and proxy...

It turns turns out that simple `make deploy` during dev loses out on the proxy config with env (HTTP_PROXY, HTTPS_PROXY, and NO_PROXY) even though seed's `HasProxy` variable reports it correctly. This is a mismatch? 

@donpenney helped me figure it out but is this something we should document OR even better add a tiny warning somewhere to note this? 

This is obviously super dev specific (it's always a bundle for customer?) issue but would nice to link some docs explaining where these vars are inject from + this gotcha! wdyt?

Reply:
We should add some checks in a couple of places, in a separate PR... if proxy is configured but LCA doesn't have the corresponding env vars set, it would mean LCA isn't really running with proxy, presumably due to dev install method:
- Reject seedgen attempt outright, as a system validation check
- LCA init can generate an error log. Possibly a health check, but maybe reject IBU attempt completely as well